test/take.js

```diff
@@ -39,4 +39,19 @@ describe('take', function() {
     eq(R.into([], R.take(0), [1, 2, 3]), []);
   });
 
+  it('steps correct number of times', function() {
+    var takeXTimes = function(x) {
+      var counter = 0;
+      R.into(
+        [],
+        R.compose(R.map(function(x) { counter += 1; return x; }), R.take(x)),
```

Comment:
running `into` for side-effects and the side-effect inside `map` is not a good example of library usage. is there a more idiomatic way to illustrate this?